### ISSUE TITLE
Dropdown: built-in filtering input element doesn't have a label

### ISSUE THREAD

**Issue by HellinSunWTW** [primefaces/primeng#14970]
### Describe the bug

The Axe Dev Tools for Accessibility is reporting the following issue:
![image](https://github.com/primefaces/primeng/assets/152492138/958a994e-871d-4d9b-a469-955c82705abf)
The input form doesn't have a aria label.
Could you add the required aria label as something like "Filtered by"

### Environment

Windows 10

### Angular version

17.2.1

### PrimeNG version

17.7.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.18.0

### Steps to reproduce the behavior

1. Install the Axe Dev Tools for Accessibility plug-in.
2. Visit the PrimeNG dropdown page (https://primeng.org/dropdown) on Chrome/Edge browser.
3. Scan the page with Axe

### Expected behavior

The required aria label gets added to the input form

**Comment by cetincakiroglu** [comment]
Hi,

There is an input property for that. You can use `ariaFilterLabel` to define an aria-label for the filter input element. 

Closing the issue.
